rpcs3/rpcs3qt/qt_utils.cpp

```diff
@@ -625,13 +625,18 @@ namespace gui
 		{
 			usz byte_unit = 0;
 			usz divisor = 1;
+#if defined(__APPLE__)
```

Comment:
If this is is used anywhere other than UI then it should be standardized. Why not just have the "ibi" variants as standard and keep binary sizes? Even macOS users will see that GiB is not GB as their OS GUI reports.

Reply:
My concern is that depending on when a log is from, the old units are meaningless as they could mean one thing or another. The same units on macOS now represent different things before and after this change.
If it's restricted to only a UI dialog or something then it's probably fine since logs aren't submitted with those.